app/facades/websocket/WebSocketController.scala

```diff
@@ -162,6 +169,24 @@ class WebSocketController @Inject() (actorSystem:  ActorSystem,
     Ok(json)
   }
 
+  private def createHandshakeResponse(ci: ConnectionInfo) = {
+    val localKeys = keys
```

Comment:
Seems redundant but not a big deal

Reply:
It is kind of 'alise' in order to be more meaningful name.